redhat-cop/casl-ansible
# Updated openshift-applier version and some IaC clean-up

#### What does this PR do?
- Updated the [openshift-applier](https://github.com/redhat-cop/openshift-applier) version to use the latest `v2.0.2` for CASL requirements. 
- Cleaned up some of the IaC sample inventory as it's now part of the openshift-applier repo's tests instead.

#### How should this be manually tested?
Run a CASL provisioning and observe that it works with no errors related to `openshift-applier` or `infra-ansible.

#### Is there a relevant Issue open for this?
N/A

#### Who would you like to review this?
cc: @redhat-cop/casl
